docs/installation.rst

```diff
@@ -0,0 +1,102 @@
+.. _aiapy-installing:
```

Comment:
As I've said, I don't like having an exact copy of what is already in the sunpy docs. I think this page should just point to the sunpy install page.

Reply:
Tweaked.